src/components/Email/index.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
```

Comment:
I don't think this stops the bots, but definitely worsens the experience. IMHO for reversion, we don't have a spam problem we'd need to mitigate this way.

Reply:
removed